plugins/modules/cloudfront_info.py

```diff
@@ -327,7 +327,9 @@ def list_origin_access_identities(self):
 
             if len(origin_access_identity_list['Items']) > 0:
```

Comment:
I guess here we could drop the entire if statement as origin_access_identity_list['Items'] will always fall back to an empty list (see line above).